packages/encryption/src/types.ts

```diff
@@ -2,3 +2,9 @@ export type ResponseType = {
   data: string | null;
   error: string | null;
 }
+
+export type CipherOptionType = {
+  iv?: CryptoJS.lib.WordArray | undefined;
+  format?: any ;
```

Comment:
Can we avoid any and have a proper type?

Reply:
The library itself uses any . So cannot be fixed